src/java.base/share/classes/java/util/GregorianCalendar.java

```diff
@@ -1741,8 +1727,7 @@ public int getActualMaximum(int field) {
 
         int value = -1;
         switch (field) {
-        case MONTH:
-            {
+            case MONTH -> {
```

Comment:
HI @amaembo, thanks for taking a look at this.

I think you should be careful here, and ask is introducing the enchanced switch adding any value? While I think the enchanced switch can be valuable when it makes the code more readable, it shouldn't be introduced just for the sake of using it.

Reply:
In this particular place, it makes the code more readable in the following points:
- It removes `break;` operators at the end of each branch, which add nothing but a visual noise
- It makes the whole switch shorter by 22 lines
- Using `->` syntax clearly states that no branch in this switch has a fall-through behavior, so the code reader should not check every branch to ensure this. Just see very first `->` and you already know that no fallthrough is here.
- In case if new branches will appear in the future, the new-style switch will protect future maintainers from accidental fall-through, an error that often happens with old-style switches.